### ISSUE TITLE
System.Reflection.ReflectionTypeLoadException: Unable to load one or more of the requested types.

### ISSUE THREAD

**Issue by george-ckito** [NuclearPowered/Reactor.Impostor#4]
### Unable to load one or more of the requested types.
I've just installed Impostor v1.8.0 and copied Reactor.Impostor.dll to servers plugin folders. After launching the exe, I get the following error:
```
System.Reflection.ReflectionTypeLoadException: Unable to load one or more of the requested types.
Method 'HandleMessageAsync' in type 'Reactor.Impostor.Net.Message255Reactor' from assembly 'Reactor.Impostor, Version=2.0.0.0, Culture=neutral, PublicKeyToken=null' does not have an implementation.
Method 'HandleRpcAsync' in type 'Reactor.Impostor.Net.Rpc255Reactor' from assembly 'Reactor.Impostor, Version=2.0.0.0, Culture=neutral, PublicKeyToken=null' does not have an implementation.
```

**Comment by miniduikboot** [comment]
I posted a PR that fixes this issue: #3 , can you check if that fixes the issue for you?

**Comment by george-ckito** [comment]
It doesn't look like It's been build yet, should I clone this and build it myself?

**Comment by js6pak** [comment]
Fixed in 31751cf4dc51fa7622242a81e3dd3d34d3e1216c
